src/qt/trafficgraphwidget.h

```diff
@@ -22,22 +24,22 @@ class TrafficGraphWidget : public QWidget
 public:
     explicit TrafficGraphWidget(QWidget *parent = nullptr);
     void setClientModel(ClientModel *model);
-    int getGraphRangeMins() const;
+    std::chrono::minutes getGraphRange() const;
 
 protected:
     void paintEvent(QPaintEvent *) override;
 
 public Q_SLOTS:
     void updateRates();
-    void setGraphRangeMins(int mins);
+    void setGraphRange(std::chrono::minutes mins);
```

Comment:
Maybe
```suggestion
    void setGraphRange(std::chrono::minutes new_range);
```
?

Reply:
I think `new_range` is a more suitable name for the argument, and now its unit is cleared by its type (i.e., minutes)

Addressed in f7a19ef774ef92ce348215593e3590a750c345e1